modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/FieldFeedback.tsx

```diff
@@ -15,30 +15,25 @@ export function FieldFeedback({
 	warningMessage,
 	...otherProps
 }: IProps) {
-	if (!errorMessage && !helpMessage && !warningMessage) {
-		return null;
-	}
+	const message =
+	warningMessage && !errorMessage ? warningMessage : errorMessage;
```

Comment:
I'm probably missing something, but why do we need `&& !errorMessage` here? Would it be simpler for it to be `warningMessage || errorMessage`?